stan-dev/cmdstan
# Use Github Actions for manual build test

#### Submisison Checklist


#### Summary:

Fixes #884 by removing Circle CI and replacing it with Github Actions. If this is approved we also need to remove the Circle CI webhook in the repo settings. Otherwise Circle CI test fails as this PR removes the test yml file.

How to review: 

- check https://github.com/stan-dev/cmdstan/pull/883/commits/6351b31f593a5eccdde8438208719945687c0230 that the Circle CI passed even with error
- see that Github actions failed on the same error as it should: https://github.com/stan-dev/cmdstan/runs/726799350
- check the Github Actions setup. Its really intuitive, so no experience needed there
- see that test pass if error removed and the manual PDF is built on success: https://github.com/stan-dev/cmdstan/actions/runs/121292187

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Rok Češnovar, Univ. of Ljubljana

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
